changelog/upgrade-notes.md

```diff
@@ -1,5 +1,7 @@
 ## Upgrade notes
 
+### v3.14.0
+
```

Comment:
I've followed the [Release Procedure](https://github.com/openlayers/ol3/wiki/Release-Procedure) but I agree that it's not necessary